frameworklauncher/src/main/java/com/microsoft/frameworklauncher/zookeeperstore/ZooKeeperClient.java

```diff
@@ -143,19 +147,50 @@ public void deleteRecursively(String path, Boolean childrenOnly) throws Exceptio
   // Set/Get small size (<= ZK_MAX_NODE_BYTES) yaml object to the node of the given path, no matter the given path exist or not.
   // DISTRIBUTED THREAD SAFE
   public <T> void setSmallYamlObject(String path, T yamlObject) throws Exception {
```

Comment:
>setSmallYamlObject [](start = 18, length = 18)

Rename to more generic setSmallObject?

Reply:
OK

---
In reply to: [170518311](https://github.com/Microsoft/pai/pull/338#discussion_r170518311) [](ancestors = 170518311)